### PR TITLE
Can now manually disable prev/next button in Pagination component

### LINKED ISSUE #150: Disable next button on Pagination when making data fetch
Currently, the previous and the next buttons are disabled when the user is on the first or last page.

Is it possible to disable the button when the user clicks NEXT so they don't have the option to continuously click through and make a bunch of async requests?  Thanks

demo

### PR DESCRIPTION
**Pull request by MrCreeper1008**
A prop `disabled: PropTypes.bool` is added to `<Pagination />` which disables all buttons in pagination.
Closes #150.

### CODE REVIEW

**Comment by couds** [comment]
Hi @MrCreeper1008 Thanks for this, but can you cange it to a prop `disabled` that will disable all navigation, because if not will be quite difficult to handle also the specific page button.

**Comment by MrCreeper1008** [comment]
At first I thought it would be great if devs were given finer controls over what buttons could be disabled, but I figure it wasnt all that useful. so @couds you're right - a disabled prop should do the job.

File: src/components/pagination/__test__/pagination.test.js [hunk 1]
```diff
@@ -27,6 +27,14 @@ describe('Pagination component', () => {
     const component = renderer.create(<Pagination total={3} current={3} />);
     expect(component.toJSON()).toMatchSnapshot();
   });
+  it('All buttons should be disabled', () => {
+    const component = renderer.create(<Pagination total={3} current={1} disabled />);
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+  it('previous button should be disabled when the prop prevDisabled is given', () => {
+    const component = renderer.create(<Pagination prevDisabled />);
```
Comment (couds) [inline src/components/pagination/__test__/pagination.test.js]: This test its not valid anymore right?

Comment (MrCreeper1008) [reply src/components/pagination/__test__/pagination.test.js]: Yeah it shouldn't be valid anymore; not sure why the tests passed. Will fix this asap.

**Pull request merged**